src/constants.ts

```diff
@@ -1,6 +1,10 @@
-export enum Page {
-  home = "home",
-  myMovies = "myMovies"
+export enum Paths {
```

Comment:
Nit: it's more idiomatic to name this `Path` since using the type as `activePath?: Paths | null;` looks a bit weirder than `activePath?: Path | null;`